sable_network/src/history/mod.rs

```diff
@@ -22,3 +26,7 @@ impl HistoryItem for HistoryLogEntry {
         &self.details
     }
 }
+
+/// A more concrete representation of [`HistoryItem`], with all its fields inflated
+/// to strings that will be sent to the client
+pub enum HistoryMessage {}
```

Comment:
Doesn't seem to be used anywhere?

Reply:
indeed, I replaced it with `HistoricalEvent`